pkg/install/install_test.go

```diff
@@ -39,12 +61,22 @@ func TestFillInTemplatesAllParameters(t *testing.T) {
 		ManifestGeneration: true,
 		AdditionalFluxArgs: []string{"arg1=foo", "arg2=bar"},
 		RegistryScanning:   true,
-	})
+		NoSecurityContext:  false,
+	}
+
+	manifests := testFillInTemplates(t, 6, params)
+
+	memDeploy := manifests["memcache-dep.yaml"]
+	deployment := testExtractDeployment(t, memDeploy)
 
+	container := deployment.Spec.Template.Spec.Containers[0]
+	if container.SecurityContext == nil {
```

Comment:
Please create a separate test for this `TestFillInTemplatesAllParameters` is supposed to only do that

Reply:
Good point, i have created this test